code/__DEFINES/~skyrat_defines/traits.dm

```diff
@@ -60,3 +60,6 @@
 #define TRAIT_XENO_ABILITY_GIVEN "xeno_ability_given"
 /// Determines if something can receive healing from a xeno
 #define TRAIT_XENO_HEAL_AURA "trait_xeno_heal_aura"
+
+/// Whether something is repairable by the anvil
+#define TRAIT_ANVIL_REPAIR (1<<11)
```

Comment:
make our own version of obj_flags and move all the checks to that (similar to `flags` and `flags_1`), we don't need the headache of 1<<11 getting overridden down the road

Reply:
I dislike the idea of adding a whole variable to every. single. obj. but if you want it...